src/navigator/QRNavigator.test.tsx

```diff
@@ -46,6 +57,25 @@ function getProps(qrCodeStyle: QRCodeStyle): QRCodeProps {
 }
 
 describe('QRNavigator', () => {
+  beforeAll(async () => {
+    await Statsig.initialize('client-sdk-key-testing', null, { localMode: true })
+  })
+  afterEach(async () => {
+    const overrides = Statsig.getAllOverrides()
+    Object.keys(overrides.gates).forEach((gateName) => Statsig.removeGateOverride(gateName))
+    Object.keys(overrides.configs).forEach((configName) => Statsig.removeConfigOverride(configName))
+    Object.keys(overrides.layers).forEach((layerName) => Statsig.removeLayerOverrie(layerName)) // lol @ the typo, I (Charlie) submitted a PR to fix their SDK but it will take time to get a new version published
```

Comment:
:facepalm:

Reply:
they at least merged my PR

Reply:
I love that Statsig is open source so I can do stuff like that instead of just report and wait

Reply:
they don't have CD set up to publish new NPM versions though sadly